src/mixxx.cpp

```diff
@@ -561,6 +563,8 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
 
     m_pOscClientManager = new OscClientManager(pConfig, m_pEngine);
 
+    m_pOscServer = new OscServer(pConfig);
```

Comment:
@daschuer That makes sense and I understand that unique_ptr is safer. However, may I ask why we would use unique_ptr since the rest of the codebase seems to use new/delete (e.g. line 181 where m_pSettingsManager = new SettingsManager)? Also, would it be preferable to just store an actual OscClientManager instance rather than a pointer to it in the class? I've always been a little confused why all of the manager members of MixxxMainWindow were stored as pointers.

Reply:
Also, sorry I've been gone for so long